azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeTelecomIntegration.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+import com.azure.android.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * CallCompositeTelecomIntegration {@link CallCompositeTelecomIntegration}.
+ */
+public final class CallCompositeTelecomIntegration extends ExpandableStringEnum<CallCompositeTelecomIntegration> {
```

Comment:
What is default behavior if not SDK Provided or Application Provided?

Can there be a default value for the config for the "unspecified" option, or are one of these the default?

Is this class intended to be temporary once calling SDK GA's? May be a good use of Compile time flags/structure to hide this SDK so it's auto-chosen on version if this it the intention.

Reply:
As per discussion we will fix for GA.